running-your-first-ot-server/setup-webserver-linux.md

```diff
@@ -0,0 +1,214 @@
+# Getting started
+Rent a Ubuntu Server 18.04 or higher from a trusted hosting company or install for the sake of learning experience Ubuntu Server 18.04 or higher in VirtualBox.
+
+Once you're set we will start by accessing our server. If you're on windows you should get putty(https://www.putty.org/), if you're on macOS or Linux you can use the terminal which is standard available. With putty just follow the screen ( insert your domain/ip-address and click ok ) for macOS and linux users type:
```

Comment:
Lot's of typos:
* `PuTTY` not `putty`
* No spaces inside `(` and `)`
* `IP address` not `ip-address`
* `click "OK"` not `click ok`
* `Linux` not `linux`

Reply:
You can use a link to PuTTy instead